src/server/services/service/about/about.test.js

```diff
@@ -4,31 +4,43 @@ import {
   mockCommonServicesCalls,
   mockServicesAdditionalCalls
 } from '~/test-helpers/common-page-rendering.js'
+import { scopes } from '~/src/server/common/constants/scopes.js'
+import { availableMigrationsFixture } from '~/src/__fixtures__/migrations/available-migrations.js'
 
 jest.mock('~/src/server/common/helpers/fetch/fetch-tenant-service.js')
 jest.mock('~/src/server/common/helpers/fetch/fetch-running-services.js')
 jest.mock('~/src/server/services/helpers/fetch/fetch-repository.js')
 jest.mock('~/src/server/services/helpers/fetch/fetch-vanity-urls.js')
 jest.mock('~/src/server/services/helpers/fetch/fetch-api-gateways.js')
-jest.mock(
-  '~/src/server/deploy-service/helpers/fetch/fetch-available-versions.js'
-)
 jest.mock('~/src/server/common/helpers/fetch/fetch-json.js')
 jest.mock('~/src/server/common/helpers/fetch/fetch-entities.js')
 jest.mock('~/src/server/common/helpers/auth/get-user-session.js')
+jest.mock('~/src/server/services/helpers/fetch/fetch-available-migrations.js')
+jest.mock('~/src/server/common/helpers/fetch/fetch-latest-migrations.js')
+jest.mock(
+  '~/src/server/deploy-service/helpers/fetch/fetch-available-versions.js'
+)
 
 describe('About Service page', () => {
   /** @type {import('@hapi/hapi').Server} */
   let server
 
   beforeAll(async () => {
+    jest.useFakeTimers({ advanceTimers: true })
+    jest.setSystemTime(new Date('2025-05-10T14:16:00.000Z'))
```

Comment:
What are the timers actually used for? cant we just mock the responses with date times in?

Reply:
relative dates rendered from dates in fixtures:
- `1pm today`,
- `1pm yesterday`,
- `1pm last saturday`